teamC/JeongDayoung/app.js

```diff
@@ -0,0 +1,40 @@
+dotenv.config();
+// Build-in package
+const http = require("http");
+
+// 3rd-party package
+const express = require("express");
+const morgan = require("morgan");
+const cors = require("cors");
+const dotenv = require("dotenv");
+const { DataSource } = require("typeorm");
+
+const app = express();
+app.use(express.json());
+app.use(morgan('dev'));
+
+const myDataSource = new DataSource({
+  type : process.env.TYPEORM_CONNECTION,
+  host : process.env.TYPEORM_HOST,
+  port : process.env.TYPEORM_PORT,
+  username : process.env.TYPEORM_USERNAME,
+  password : process.env.TYPEORM_PASSWORD,
+  database : process.env.TYPEORM_DATABASE
+}) 
+
+myDataSource.initialize()
+  .then(() => {
+    console.log("Data Source has been initialized!");
+  });
+
+app.get('/ping', cors(), function(req, res, next){
```

Comment:
```suggestion
app.get('/ping', function(req, res, next){
```
cors를 /ping 엔드포인트에만 적용되기보다는 위의 리뷰처럼 middleware로서 추가하여 들어오는 모든 요청에 대해 통과 될 수 있게 해주세요~